app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<header class="container d-flex justify-content-between  align-items-center my-3">
```

Comment:
Your design is good but please try to stick to the suggested Behance design. Please add the corresponding CSS properties to make the style of your app look like the suggested.

Check the image below for a better understanding: 
![Capture2](https://user-images.githubusercontent.com/75709904/149640571-377c79b7-9e34-440f-8959-00377eb36a58.PNG)